src/Campaigns/Migrations/MigrateFormsToCampaignForms.php

```diff
@@ -38,57 +38,99 @@ public static function timestamp(): int
      */
     public function run()
```

Comment:
Let's add a `@throws Exception` to the doc block.

Reply:
Resolved, see [7ec7b59](https://github.com/impress-org/givewp/pull/7532/commits/7ec7b59f4808b86f05b30fb08e98f64e1ecf8cb6).